ansible_collections/arista/avd/roles/eos_designs/docs/tables/platform-families.md

```diff
@@ -0,0 +1,621 @@
+          enabled: <bool; default=True>
+
+          # Minimum number of power supplies required for the device. Set to 0 to skip validation.
+          min_power_supplies: <int>
+
+          # Minimum number of fans required for the device. Set to 0 to skip validation.
+          min_fans: <int>
+
+          # Minimum number of supervisor modules required for the device. Set to 0 to skip validation.
+          min_supervisors: <int>
+
+          # Minimum number of line cards required for the device. Set to 0 to skip validation.
+          min_line_cards: <int>
+
+          # Minimum number of fabric cards required for the device. Set to 0 to skip validation.
+          min_fabric_cards: <int>
+
+          # List of approved transceiver manufacturers for the device.
+          transceiver_manufacturers: # (1)!
+            - <str>
+
+        # Custom structured config for eos_cli_config_gen.
+        structured_config: <dict>
+
+        # EOS CLI rendered directly on the root level of the final EOS configuration.
+        raw_eos_cli: <str>
+
+    # Platform settings. The first entry found where the `platform` node setting is fully matched by any regex in the `platforms` list will be chosen. If no matches are found, the first entry containing a platform `default` will be chosen. The default values will be overridden if `platform_families` is defined. If you need to replace all the default platforms, it is recommended to copy the defaults and modify them. If you need to add custom platforms, create them under `custom_platform_families`. Entries under `custom_platform_families` will be matched before the equivalent entries from `platform_families`.
+    platform_families:
+
+        # Platform family name.
+      - name: <str; required; unique>
+
+        # Only applied when evpn_multicast is true.
+        trident_forwarding_table_partition: <str>
+        reload_delay:
+
+          # In seconds.
+          mlag: <int; 0-86400>
+
+          # In seconds.
+          non_mlag: <int; 0-86400>
+        tcam_profile: <str>
+        lag_hardware_only: <bool>
+
+        # Default interface MTU configured on EOS under "interface defaults".
+        # Takes precedence over the root key "default_interface_mtu".
+        default_interface_mtu: <int; 68-65535>
+
+        # Set MTU on point to point uplink interfaces.
+        # Takes precedence over the root key "p2p_uplinks_mtu".
+        # <node_type>.uplink_mtu -> platform_settings.p2p_uplinks_mtu -> p2p_uplinks_mtu -> 9214.
+        p2p_uplinks_mtu: <int; 68-65535>
+
+        # Use EOS CLI to configure kernel forwarding ECMP programming.
+        # For EOS kernel forwarding, ECMP programming can be enabled in two different ways, depending on the EOS version.
+        # - For newer EOS versions (starting 4.33.2) use the proper CLI.
+        # - For older EOS versions use an agent environment variable. Changing this requires restarting the KernelFib agent.
+        kernel_ecmp_cli: <bool; default=True>
```

Comment:
I think this should be under feature support. @gmuloc ?

Reply:
but where will it go? :p